### PR TITLE
Adding unit test cases for invites

### PR DESCRIPTION
**Pull request by kushalmalani**
fix astronomer/issues#1343

### CODE REVIEW

File: src/resolvers/types/invite/index.unit.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { role } from "./index";
+import { WORKSPACE_ADMIN } from "constants";
+
+describe("role", () => {
+  test("should return default role if no parent role is specified", async () => {
+    const parent = {
+      hello: "world"
+    };
+    const user_role = await role(parent);
```
Comment (schnie) [inline src/resolvers/types/invite/index.unit.test.js:9]: Just for consistency, what do you think about `camelCase` for names?

File: src/resolvers/query/inviteTokens/index.js [hunk 2]
```diff
@@ -9,8 +7,24 @@ import { inviteQuery } from "../../../lib/invites";
...
+}
+
+/*
+ * Build the invites query based on args.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return {Object} The users query.
+ */
+export function inviteTokensQuery(args) {
+  // Pull out some args.
```
Comment (adam2k) [inline src/resolvers/query/inviteTokens/index.js:23]: This comment can probably be deleted.

File: src/resolvers/query/inviteTokens/index.js [hunk 3]
```diff
@@ -9,8 +7,24 @@ import { inviteQuery } from "../../../lib/invites";
...
+  const { email } = args;
+
+  // If we have an email use it.
```
Comment (adam2k) [inline src/resolvers/query/inviteTokens/index.js:26]: Same here

File: src/resolvers/types/invite/index.unit.test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    const parent = {
+      hello: "world"
+    };
+    const userRole = await role(parent);
+    expect(userRole).toBe(WORKSPACE_ADMIN);
+  });
+
+  test("should return parent role if parent role is specified", async () => {
+    const parent = {
+      role: "admin"
```
Comment (adam2k) [inline src/resolvers/types/invite/index.unit.test.js:15]:
```Suggestion
   role
```

File: src/resolvers/types/invite/index.unit.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  test("should return default role if no parent role is specified", async () => {
+    const parent = {
+      hello: "world"
+    };
+    const userRole = await role(parent);
+    expect(userRole).toBe(WORKSPACE_ADMIN);
+  });
+
+  test("should return parent role if parent role is specified", async () => {
+    const parent = {
```
Comment (adam2k) [inline src/resolvers/types/invite/index.unit.test.js:14]:
```Suggestion
    const role = "admin";
    const parent = {
```

File: src/resolvers/types/invite/index.unit.test.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+      role: "admin"
+    };
+    const userRole = await role(parent);
+    expect(userRole).toBe("admin");
```
Comment (adam2k) [inline src/resolvers/types/invite/index.unit.test.js:18]:
```Suggestion
    expect(userRole).toBe(role);
```

File: src/routes/v1/registry/events/index.unit.test.js [hunk 7]
```diff
@@ -251,14 +251,15 @@ describe("POST /registry-events", () => {
...
-        $fragment: function() {
-          return null;
-        }
-      });
+    deployment = {
+      findOne: jest.fn().mockReturnValue({}),
+      update: jest.fn().mockReturnValue({
+        workspace: { id: casual.uuid },
+        label: casual.word,
+        id: casual.uuid
```
Comment (adam2k) [inline src/routes/v1/registry/events/index.unit.test.js:259]: Is this supposed to match the workspace id?  If so, I'd suggestion doing something like `const id = caual.uuid` and then delete `: casual.uuid` from both the workspace.id and id values.

Comment (adam2k) [reply src/routes/v1/registry/events/index.unit.test.js:259]: Same with `label: casual.word,`.  Instead you could do `const label = casual.word;` and then `label` in the `mockReturnValue`.

File: src/routes/v1/registry/events/index.unit.test.js [hunk 8]
```diff
@@ -325,14 +329,15 @@ describe("POST /registry-events", () => {
...
-      .mockName("deployment")
-      .mockReturnValue({
-        $fragment: function() {
-          return { config: null };
-        }
-      });
+    deployment = {
+      findOne: jest.fn().mockReturnValue({}),
+      update: jest.fn().mockReturnValue({
+        workspace: { id: casual.uuid },
```
Comment (adam2k) [inline src/routes/v1/registry/events/index.unit.test.js:335]: Same comment as above.

File: src/routes/v1/registry/events/index.unit.test.js (hunk 7 above)
Comment (kushalmalani) [reply src/routes/v1/registry/events/index.unit.test.js:259]: Strangely, i didnt write these tests. Somehow on rebasing with prisma2, they show up in my diff. Tried rebasing, and yet still see it.

Comment (adam2k) [reply src/routes/v1/registry/events/index.unit.test.js:259]: Yeah, no problem if you want to leave as-is.  It was just a suggestion to make it more readable.

File: src/routes/v1/registry/events/index.unit.test.js [hunk 9]
```diff
@@ -26,7 +26,11 @@ jest.mock("deployments/config", () => {
 describe("POST /registry-events", () => {
   let deployment = jest.fn();
   let dockerImage = jest.fn();
+  let id, label, workspace;
```
Comment (adam2k) [inline src/routes/v1/registry/events/index.unit.test.js:29]: 🎉🚀👍

**Pull request merged**